scripts/modules/MsIndustryLinks/templates/data_source/DataSourceWorkflow.psm1

```diff
@@ -269,8 +269,8 @@ function Get-ConnectorIdentifiers {
             grant_type    = 'client_credentials'
         }
 
-        $authResponse = Invoke-RestMethod -Method Post -Uri $tokenUrl -Headers $authHeaders -Body $authBody -ErrorAction Stop
-        $accessToken = $authResponse.access_token
+        $authResponse = Invoke-WebRequest -Method Post -Uri $tokenUrl -Headers $authHeaders -Body $authBody -ErrorAction Stop
```

Comment:
Why does this need changing to `Invoke-WebRequest`?

Reply:
Invoke-RestMethod doesn't auto-convert the body into the correct content-type.